server/odc-service/src/main/java/com/oceanbase/odc/service/structurecompare/StructureComparisonService.java

```diff
@@ -147,6 +148,12 @@ public List<StructureComparisonTaskResultEntity> batchCreateTaskResults(List<DBO
     }
 
     private void checkPermission(@NonNull StructureComparisonTaskEntity taskEntity) {
+        if (taskEntity.getFlowInstanceId() == null) {
```

Comment:
structure compare task is bind to flow instance id, the FlowInstanceId would not be null

Reply:
okay~ I removed it.